server/db_mysql.go

```diff
@@ -129,6 +131,122 @@ func (ms *MsqlCache) NextFixity(cutoff time.Time) string {
 	return item
 }
 
+// GetFixityById
+func (ms *MsqlCache) GetFixityById(id string)  *fixity {
+        const query = `
+                SELECT  id, item, scheduled_time, status, notes
+                FROM fixity
+                WHERE id = ?
+                LIMIT 1`
+
+	var thisFixity = new(fixity)
+	var thisWhen mysql.NullTime
+
+        err := ms.db.QueryRow(query, id).Scan(&thisFixity.Id, &thisFixity.Item, &thisWhen, &thisFixity.Status, &thisFixity.Notes)
+        if err == sql.ErrNoRows {
+                // no next record
+                return nil
+        } else if err != nil {
+                log.Println("GetFixtyByID  MySQL queryrow", err.Error())
+                return nil
+        }
+
+        // Handle for nil time value
+	if thisWhen.Valid {
+	    thisFixity.Scheduled_time = thisWhen.Time
+	} else {
+	    thisFixity.Scheduled_time = time.Time{}
```

Comment:
This line is unnecessary.

Reply:
I'm not sure which line you mean- 157?

Reply:
Nevermind- saw your comment for line 193.